vuestorefront/storefront-ui
# fix: SfAccordion - memory leak

# Related issue
Closes #2143 

# Scope of work
- removed mutating $children directly

# Screenshots of visual changes
-
# Testing instructions:
- start production env on nuxt
- switch between category and homepage
- check performance monitor
